accesstoken.go

```diff
@@ -59,6 +59,10 @@ func (c *WorkwxApp) accessTokenRefresher() {
 	}
 }
 
+func (c *WorkwxApp) SyncAccessToken() error {
+	return c.syncAccessToken()
```

Comment:
How is this useful, given there is already mechanism built using separate goroutine?

Reply:
removed